src/hmem.c

```diff
@@ -208,15 +208,15 @@ ssize_t ofi_copy_to_hmem_iov(enum fi_hmem_iface hmem_iface, uint64_t device,
 				     (void *) src, size, OFI_COPY_BUF_TO_IOV);
 }
 
-int ofi_hmem_get_handle(enum fi_hmem_iface iface, void *dev_buf, void **handle)
+int ofi_hmem_get_handle(enum fi_hmem_iface iface, void *dev_buf, size_t size, void **handle)
```

Comment:
Here and several other places, please limit line lengths to about 80 characters. A lot of the hmem related code are too long.